lib/active_merchant/billing/gateways/card_stream.rb

```diff
@@ -283,6 +284,20 @@ def add_threeds_required(post, options)
         add_pair(post, :threeDSRequired, options[:threeds_required] || @threeds_required ? 'Y' : 'N')
       end
 
+      def add_threeds_fields(post, options)
+        return unless three_d_secure = options[:three_d_secure]
+
+        add_pair(post, :threeDSEnrolled, three_d_secure[:enrolled] == 'true' ? 'Y' : 'N')
```

Comment:
I believe the value for `three_d_secure[:enrolled]` is likely to already be `Y` or `N` rather than a boolean - we should account for both cases. See Cristian's pattern here: https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/d_local.rb#L309-L315